src/app/[locale]/specs/lexicon/en.mdx

```diff
@@ -208,6 +229,30 @@ Note that unlike `object`, there is no `nullable` field on `params`.
 
 `params` can not be declared as a top-level named type in a schema `defs` array. This means that they can not be referenced by a `ref` or `union`.
 
+### `permission`
+
+This is a limited-scope type which is only ever used for the `permissions` field of the `permission-set` primary type, and should not appear in other schema contexts. They map to [auth permissions](/specs/permission).
+
+Type-specific fields:
+
+- `resource` (string, required): indicates the resource type which access is being granted to
+
+Additional `permission` fields depend on the resource type being declared. The set of resources and fields may expand over time, but currently includes:
+
+- `repo`: public repository write permissions
+    - `collection` (array of unique strings, required): indicates record types. Must include at least one element (not empty array). Does not support wildcard (`*`) in Lexicon schema context.
+    - `action` (array of unique strings, optional): limits set of record operations. Values must be one or `create`, `update`, `delete`, case-sensitive.
+- `rpc`: remote API calls via proxying or service token generation
+    - `lxm` (array of strings, required): allowed XRPC endpoints, as NSIDs. Must include at least one element (not empty array). Wildcard (`*`) is not supported in Lexicon schema context.
+    - `aud` (string, semi-required): remote service ("audience"). In the context of permission sets, can not have fixed DID service references. Supports wildcard (`*`), though `aud` and `lxm` can not both be wildcard. Required unless `inheritAud` is true. Effectively must be either wildcard, or undefined with `inheritAud` set to true.
+    - `inheritAud` (boolean, optional): if true, `aud` does not need to be defined.
+
+Note that the `blob`, `account`, and `identity` resources types are not supported in Lexicon schema context.
+
+Permission declarations with unknown resource types or unknown parameters (fields) must be ignored by services implementing access control. This is because the set of permissions is expected to evolve over time, and additional fields might attenuate the permission granted. It is unsafe to ignore unknown fields when parsing permissions for access control.
```

Comment:
We should make it very explicit that `blob`, `account`, and `identity` must be ignored if present. Either like suggested here, or as an additional sentense in the previous paragraph.

```suggestion
Permission declarations with unknown or disallowed resource types or unknown parameters (fields) must be ignored by services implementing access control. This is because the set of permissions is expected to evolve over time, and additional fields might attenuate the permission granted. It is unsafe to ignore unknown fields when parsing permissions for access control.
```